sentry-android-core/src/main/java/io/sentry/android/core/AnrIntegration.java

```diff
@@ -44,6 +44,11 @@ private void register(IHub hub, SentryAndroidOptions options) {
 
   @TestOnly
   void reportANR(IHub hub, final @NotNull ILogger logger, ApplicationNotResponding error) {
+    if (!hub.isIntegrationEnabled(AnrIntegration.class)) {
+      logger.log(SentryLevel.WARNING, "AnrIntegration is not enabled to the current hub.");
```

Comment:
Not sure WARN is a good choice here since it's a legit case.
The API allows you to bind a new client to the hub and capture errors. It's intentional that integrations that are not set to that hub would not run in that case.

Can we have?:
```suggestion
      logger.log(SentryLevel.INFO, "AnrIntegration is not enabled to the current hub.");
```

Reply:
makes sense, thanks